### ISSUE TITLE
Not pulling in latest version (2.1.0) from NPM?

### ISSUE THREAD

**Issue by JeffBeltran** [shentao/vue-multiselect#731]
Not sure if i'm jumping the gun but when i try to pull in the latest version from NPM it doesn't work but if i pull in via github `npm install shentao/vue-multiselect` i get the lasted version.

**Comment by shentao** [comment]
Npm states it got the 2.1.0 version. It should be working properly. Gonna check it out tomorrow. Thanks for letting me know!

**Comment by JeffBeltran** [comment]
Yeah that's why i was confused for a bit, once i took a look into node modules i noticed that the src/Multiselect.vue was missing the changes from #700. I'm not super familiar with NPM packages though so figured i should at least point it out incase it was something weird. Thanks again
